chain/storage/leveldb.cpp

```diff
@@ -50,6 +52,14 @@ ResLevelDB::ResLevelDB(std::optional<LevelDBInfo> config) {
       path = (*config).path();
     }
   }
+  if ((*config).enable_block_cache()) {
+    LRUCache<std::string, std::string>* block_cache =
```

Comment:
When to destroy this pointer?
It looks like there is a memory leak.

You can use unique_pointer instead.